config/locales/en.yml

```diff
@@ -510,6 +510,7 @@ en:
     search: Search
     security_html: Your information is secure and the information you provide will be handled in accordance with our %{privacy_link}.
     select_file: Select file
+    select_prompt: "-Select-"
```

Comment:
For a generic default, I'd remove the dashes, maybe say "Select option". My first instinct would have been to use "Select county" and "Select school district" but I'm not sure whether that would look okay visually.